node/network/src/peer.rs

```diff
@@ -17,6 +17,7 @@ use tokio::net::TcpStream;
 use tokio::prelude::stream::SplitStream;
 use tokio::timer::Delay;
 use tokio_serde_cbor::Codec;
+use std::sync::RwLockWriteGuard;
```

Comment:
parking_lot's `RwLockWriteGuard` is probably faster.

Reply:
It does not handle lock poisoning, which is not safe: https://www.reddit.com/r/rust/comments/4kng0h/parking_lot_highly_optimized_synchronization/d3gh3gt